it/mail/chrome/messenger/messenger.properties

```diff
@@ -914,30 +914,30 @@ cmd_clearHistory_accesskey=C
 imageDimensions=%1$S x %2$S
 
 # changing the locale
-pbRestartToChangeLocale.title=Change Language
+pbRestartToChangeLocale.title=Cambia lingua
 # LOCALIZATION NOTE(pbRestartToChangeLocale.msg): %1$S is replaced by the brandShortName
-pbRestartToChangeLocale.msg=You need to restart %1$S in order to change the language.
-pbResartToChangeLocale.restart=&Restart Now
-pbResartToChangeLocale.later=Restart &Later
+pbRestartToChangeLocale.msg=Devi riavviare %1$S per cambiare la lingua.
+pbResartToChangeLocale.restart=&Riavvia ora
+pbResartToChangeLocale.later=Riavvia &più tardi
 
 # pbLanguageProcess.js
-downloadingLangPack=Downloading %1$S...
-verifyingLangPack=Verifying...
-installingLangPack=Installing...
-unableToInstallLangPack=Unable to install %1$S:  %2$S.
-unableToInstallLangPackTitle=Problem Installing Language
-requestFailed=Request Failed
+downloadingLangPack=Download %1$S in corso...
```

Comment:
CORRECTION NEEDED.

I would add "di" after "Download", as follows: "Download di %1$S in corso..."